lib/Reference/ReferenceProvider.php

```diff
@@ -73,24 +73,57 @@ public function getIconUrl(): string {
 	public function matchReference(string $referenceText): bool {
 		$start = $this->urlGenerator->getAbsoluteURL('/apps/' . Application::APP_ID);
 		$startIndex = $this->urlGenerator->getAbsoluteURL('/index.php/apps/' . Application::APP_ID);
+		//public calendars  /\/p\/([a-zA-Z0-9]+)/
```

Comment:
duplication of the regex one line later?

Reply:
a bit confused here you mean `\/[a-zA-Z0-9-]+\/[a-zA-Z0-9-]+\/$/i` and `\/p\/[a-zA-Z0-9]+$/i` ? 
If you mean the code it doesn't exist on the latest commit

Reply:
if you mean the comment , was a testing residual, removed